app/ai/voice/agents/breeze_buddy/handlers/internal/end_conversation.py

```
@@ -80,19 +97,52 @@ async def end_conversation(context: TemplateContext, args, transition_to=None):
+                            result_payload["summary"] = summary
+                        else:
+                            result_payload["transcription"] = filtered_transcript
+                    else:
+                        result_payload["transcription"] = filtered_transcript
+
+                    await publish_hold_transfer_result(pub_channel, result_payload)
+                    logger.info(
+                        f"[hold_transfer] Published outbound result for call "
+                        f"{context.call_sid} with status={status}"
+                    )
+                except Exception as pub_error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)